src/NLog.Web/README.md

```diff
@@ -5,10 +5,17 @@
 
 Supported platforms:
 
-.NET 3-5-4.8
+.NET 3.5 - 4.8
 
 Useful Links
 - [Change Log](https://github.com/NLog/NLog.Web/releases)
 - [Tutorial](https://github.com/NLog/NLog/wiki/Tutorial)
 - [ASP.NET Layout Renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web)
 - [Logging Troubleshooting](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
+
+Registration of NLog.Web in the NLog.config File
```

Comment:
Think the "Useful links" should always be at the bottom. Any advice should be moved above.

Reply:
complete